test/perf-test/perf.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
```

Comment:
@jbusche and @asm582 should we consider using [kuttl](https://kuttl.dev). @dmatch01  has introduced it the quota-management branch, and I think that your test might be a better fit into that.

Reply:
Can we consider kuttl in another PR later?  Maybe once the quota-management branch is merged in, I can leverage some of the items for the perf tests?  Otherwise, if we really want kuttl now, I think we'll have to postpone for a bit until we get some more items off the our plate.

Reply:
I vote for using kuttl later.